source/drivers/csharp.txt

```diff
@@ -34,7 +32,9 @@ with MongoDB.
 Installation
 ------------
 
-``NuGet`` is the simplest way to get the driver. `MongoDB.Driver <http://www.nuget.org/packages/mongodb.driver>`__ should be used for all new projects.
+``NuGet`` is the simplest way to get the driver. Use
+`MongoDB.Driver <http://www.nuget.org/packages/mongodb.driver>`__ for
```

Comment:
Thanks for changing this to active voice.
You should only need one underscore after an external link. I think the double underscore usually refers to an anonymous hyperlink https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets

Reply:
fixing